elements/lisk-transactions/src/register_delegate.ts

```diff
@@ -48,16 +48,6 @@ export const registerDelegate = (
 	validateInputs(inputs);
 	const { username, passphrase, secondPassphrase, networkIdentifier } = inputs;
 
-	if (!username || typeof username !== 'string') {
-		throw new Error('Please provide a username. Expected string.');
-	}
-
-	if (username.length > USERNAME_MAX_LENGTH) {
-		throw new Error(
-			`Username length does not match requirements. Expected to be no more than ${USERNAME_MAX_LENGTH} characters.`,
-		);
-	}
-
```

Comment:
I see that this piece of code was removed but related tests were not updated or maybe there were no tests? Also is this change part of this PR? Why are we removing this check?

Reply:
It was tested but it was duplicated code as the validate() function already does this. So this was being checked two times.